advanced/Scripts/update.sh

```diff
@@ -47,23 +35,29 @@ is_repo() {
 	# Use git to check if directory is currently under VCS
 	echo -n ":::    Checking $1 is a repo..."
 	cd "${1}" &> /dev/null || return 1
-	git status &> /dev/null && echo " OK!"; return 0 || echo " not found!"; return 1
+	if [[ $(git status --short) ]]; then
+    echo " OK!"
+    return 0
+  else
+    echo " not found!"
+    return 1
+  fi
 }
 
 make_repo() {
 	# Remove the non-repod interface and clone the interface
 	echo -n ":::    Cloning $2 into $1..."
 	rm -rf "${1}"
-	git clone -q --depth 1 "${2}" "${1}" > /dev/null & spinner $!
+	git clone -q --depth 1 "${2}" "${1}" > /dev/null || exit 1
```

Comment:
I know what $1 and $2 are here because I'm familiar with the code/git.  Adding 2 lines of code to apply 'names' to the passed in variables is nice for readability and defining the interface for the function.

```
cloneDestination="$1"
gitRepo="$2"
...
git clone -q --depth 1 "${gitRepo}" "${cloneDestination}" > /dev/null || exit 1
```